### PR TITLE
Feature/update sqldelight

### PR DESCRIPTION
**Pull request by TalhaAli00**
This will be merged after 1.18.0

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[versions]
```
Comment (jakubuid) [inline gradle/libs.versions.toml:1]: Curious: what doest this file?

**Review by jakubuid — APPROVED** [review]
Just to double check - did you check that it doesn't break anything?  E.g. the update path

File: gradle/libs.versions.toml (hunk 1 above)
Comment (TalhaAli00) [reply gradle/libs.versions.toml:1]: documentation regarding version catalog
https://developer.android.com/build/migrate-to-catalogs

**Review by Elyniss — APPROVED** [review]
I think if we update the branch should be ready to merge

**Review by BufferAngel — COMMENTED** [review]
I have no idea what  this is?   Or why it is?  What does it do?  Who are you?  Why is this on my phone?

**Pull request merged**